packages/amplify-e2e-core/src/index.ts

```diff
@@ -31,6 +32,12 @@ export function isCI(): boolean {
   return process.env.CI && process.env.CIRCLECI ? true : false;
 }
 
+export function injectSessionToken() {
+  if (!fs.readFileSync(pathManager.getAWSCredentialsFilePath()).toString().includes('aws_session_token')) {
+    fs.appendFileSync(pathManager.getAWSCredentialsFilePath(), `aws_session_token=${process.env.AWS_SESSION_TOKEN}`);
```

Comment:
The credential file uses `ini` format. We should be using `ini` package and update the file instead doing an `appendFileSync`

Reply:
:+1: